roadmap/implementers-guide/src/node/collators/collator-protocol.md

```diff
@@ -108,14 +108,20 @@ When acting on an advertisement, we issue a `Requests::CollationFetching`. If th
 
 As a validator, once the collation has been fetched some other subsystem will inspect and do deeper validation of the collation. The subsystem will report to this subsystem with a [`CollatorProtocolMessage`][CPM]`::ReportCollator` or `NoteGoodCollation` message. In that case, if we are connected directly to the collator, we apply a cost to the `PeerId` associated with the collator and potentially disconnect or blacklist it.
 
-### Interaction with [Candidate Selection][CS]
+### Interaction with [Candidate Backing][CB]
 
-As collators advertise the availability, we notify the Candidate Selection subsystem with a [`CandidateSelection`][CSM]`::Collation` message. Note that this message is lightweight: it only contains the relay parent, para id, and collator id.
+As collators advertise the availability, we notify the Candidate Selection subsystem with a [`CandidateBacking`][CBM]`::Second` message. Note that this message contains the relay parent of the advertised collation, the candidate receipt and the [PoV][PoV].
 
-At that point, the Candidate Selection algorithm is free to use an arbitrary algorithm to determine which if any of these messages to follow up on. It is expected to use the [`CollatorProtocolMessage`][CPM]`::FetchCollation` message to follow up.
+For the moment, a validator will simply second the first valid parablock candidate per relay head. See [Future Work](#future-work).
 
-The intent behind this design is to minimize the total number of (large) collations which must be transmitted.
+## Future Work
 
+Several approaches have been discussed, but all have some issues:
+
+- The current approach is very straightforward. However, that protocol is vulnerable to a single collator which, as an attack or simply through chance, gets its block candidate to the node more often than its fair share of the time.
+- It may be possible to do some BABE-like selection algorithm to choose an "Official" collator for the round, but that is tricky because the collator which produces the PoV does not necessarily actually produce the block.
```

Comment:
We already have a de facto selection algorithm if collators produce blocks via Aura, BABE, or in future Sassafras, not sure why anyone else could submit the block for them in those cases.